test_build_functions/CMakeLists.txt

```diff
@@ -351,23 +351,27 @@ function(build_c_tests whatIsBuilding use_gballoc folder)
     endif()
 endfunction()
 
-function(set_default_build_options)
-    #use _CRT_SECURE_NO_WARNINGS by default
+macro(set_default_build_options)
     if(MSVC)
+        #use _CRT_SECURE_NO_WARNINGS by default
         add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+
+        # warning C4200: nonstandard extension used: zero-sized array in struct/union : looks very standard in C99 and it is called flexible array. Documentation-wise is a flexible array, but called "unsized" in Microsoft's docs
+        # https://msdn.microsoft.com/en-us/library/b6fae073.aspx
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4200")
```

Comment:
why not move that to section below (also if(MSVC))?

Reply:
Moved

---
In reply to: [425427511](https://github.com/Azure/azure-c-build-tools/pull/24#discussion_r425427511) [](ancestors = 425427511)